src/v3/components/user_account_item/index.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, IconButton } from '@mui/material';
+import { UserAccountItemProps } from './user_account_item.types';
+
+import { StyledBoxWrapper } from './user_account_item.styled';
+import { useState } from 'react';
+import { IconAccount, IconAdmin, IconArrowLink, IconAssignment } from '@components/icons';
+
+/**
+ * Renders user account information based on the variant.
+ * @param {Object} props - The props for the component.
+ * @param {('user' | 'admin' | 'baptized')} props.variant - The variant of the user account item.
+ * @param {string} props.userName - The name of the user.
+ * @param {string} [props.userPosition] - The position of the user (applicable for 'admin' variant).
+ * @returns {JSX.Element} - JSX element representing the user account information.
+ * @example
+ * // Renders user name for a regular user:
+ * <UserAccountItemTextContent variant="user" userName="John Doe" />
+ *
+ * // Renders user name and position for an admin user:
+ * <UserAccountItemTextContent variant="admin" userName="Admin" userPosition="Administrator" />
+ *
+ * // Renders user name for a baptized user:
+ * <UserAccountItemTextContent variant="baptized" userName="Jane Doe" />
+ */
+const UserAccountItemTextContent = (props: {
+  variant: 'user' | 'admin' | 'baptized';
+  userName: string;
+  userPosition: string;
+}) => {
+  switch (props.variant) {
+    case 'user':
+    case 'baptized':
+      return (
+        <div className="h4" style={{ fontWeight: '550' }}>
+          {props.userName}
+        </div>
+      );
+
+    case 'admin':
+      return (
+        <Box>
+          <div className="h4" style={{ fontWeight: '550', margin: '0' }}>
```

Comment:
@FussuChalice: here, I guess here you should be using the `Typography` component, instead of a `div`. All these additional styles are already included in that component when you assign it the correct `className`.

Reply:
Okay. I fixed this.